lib/awful/mouse/init.lua

```diff
@@ -297,8 +297,8 @@ function mouse.object.set_current_client() end
 
 function mouse.object.get_current_wibox()
     local obj = capi.mouse.object_under_pointer()
-    if type(obj) == "drawin" then
-        return obj
+    if type(obj) == "drawin" and obj.wibox then
+        return obj.wibox()
```

Comment:
Sorry then. Apparently I was confused. I thought this used $MAGIC to then call your `get_wibox` function which directly returned the wibox, instead of a function returning the wibox. Apparently I'm wrong....